site/styles/_variables.scss

```diff
@@ -31,7 +31,12 @@
 
   --carousel-height: 560px;
   --carousel-height-lg: 620px;
-  --s72-carousel-height: 20px; // Adds additional height to s72-carousel in carousel.scss.
+
+  // "Empty" container used to push down content below the carousel. 
+  // Carousel height - Height of anything above the carousel + 30px.
+  --s72-carousel-height: 480px;
+  --s72-carousel-height-sm: 480px; // 550 - 100 + 30
+  --s72-carousel-height-lg: 478px; // 620 - 172 + 30
```

Comment:
These are gonna need a better name lol

Reply:
Related, I've been thinking about possibly prefixing our CSS vars with `--s72` to avoid any possible naming conflicts with third parties, e.g. Intercom or anything else brought in by GTM.